pages_ru/rails/200_real_apps/080_configuration.md

```diff
@@ -0,0 +1,172 @@
+{% include snippetcut_example path=".helm/templates/deployment.yaml" syntax="yaml" snippet="nginx_conf_checksum" examples=page.examples %}
+Для каждого подключенного ConfigMap/Secret потребуется отдельная аннотация.
+>_Авторы самоучителя предпочитают вместо аннотаций с хеш-суммами использовать операторы вроде [stakater/Reloader](https://github.com/stakater/Reloader), т. к. они проще, гибче и удобнее в работе._
+
+## Конфигурация через values.yaml
+
+Для упрощения конфигурации манифестов часто используемые параметры можно выносить в `.helm/values.yaml`:
+{% include snippetcut_example path=".helm/values.yaml" syntax="yaml" snippet="mysql" examples=page.examples %}
+... и подставлять их обратно в манифест так:
+{% include snippetcut_example path=".helm/templates/database.yaml" syntax="yaml" snippet="volume_claim_templates" examples=page.examples %}
+Это также позволит иметь разные значения для разных окружений, о чем более подробно будет рассказано в следующих разделах.
+
+Но особенно полезным будет перенос в `.helm/values.yaml` повторяющейся конфигурации, например переменных окружения для приложения, которые мы используем в нескольких местах сразу:
+{% include snippetcut_example path=".helm/values.yaml" syntax="yaml" snippet="app" examples=page.examples %}
+Теперь переменные окружения из `.Values.app.envs` можно либо подставлять в манифест в `env` контейнера, как было ранее, либо вынести эти переменные окружения в ConfigMap, который потом подключить в контейнер через `envFrom`.
+
+Первый вариант проще, но вариант с ConfigMap удобнее при большом количестве общих переменных окружения, т. к. позволит избежать их дублирования между контроллерами. ConfigMap может выглядеть так:
+{% include snippetcut_example path=".helm/templates/configmap-app-envs.yaml" syntax="yaml" examples=page.examples %}
+
+Подключаем этот ConfigMap через `envFrom` в Deployment как набор переменных окружения:
+{% include snippetcut_example path=".helm/templates/deployment.yaml" syntax="yaml" snippet="backend" examples=page.examples %}
+Не забываем про аннотации с хеш-суммами, которые приведут к пересозданию Pod'ов при изменении ConfigMaps:
+{% include snippetcut_example path=".helm/templates/deployment.yaml" syntax="yaml" snippet="configmap_app_envs_checksum" examples=page.examples %}
+Аналогично этот ConfigMap подключается и в остальные ресурсы, в которых требуются эти переменные окружения.
+
+## Секреты
+
+Прежде, чем начать работу с секретами, нам потребуется ключ шифрования. В самоучителе используется готовый ключ, находящийся в файле `.werf_secret_key`, и который будет добавлен в репозиторий для того, чтобы вы могли расшифровать подготовленные нами секреты.
```

Comment:
Ok, I see, we have a warning, and `.werf_secret_key` contains `aaaaaa....` content. User will see such content and think about changing this key.

I think we need some info about generating your own key.